relay/config.go

```diff
@@ -76,23 +77,32 @@ type Config struct {
 	// ChunkMaxConcurrency is the size of the work pool for fetching chunks. Default is 32. Note that this does not
 	// impact concurrency utilized by the s3 client to upload/download fragmented files.
 	ChunkMaxConcurrency int
+
+	// MaxKeysPerGetChunksRequest is the maximum number of keys that can be requested in a single GetChunks request.
+	// Default is 1024. // TODO should this be the max batch size? What is that?
+	MaxKeysPerGetChunksRequest int
```

Comment:
@ian-shim what is the maximum number of keys we should permit in a single request?